webpack.common.js

```diff
@@ -46,7 +46,13 @@ module.exports = {
             commonjs2: 'prop-types',
             commonjs: 'prop-types',
             amd: 'prop-types',
-        }
+        },
+        jquery: {
+            root: '$',
```

Comment:
It's not `$` it's `jQuery`.

Reply:
I've taken it from: https://github.com/ezsystems/ezplatform-page-builder/blob/master/src/bundle/ui-dev/webpack.common.js#L43

Reply:
I know, but it turns out that it works by accident.
I've found out about that when I had to debug why form builder is not working in content on the fly.
https://github.com/ezsystems/ezplatform-form-builder/pull/104/files#diff-b3fe84cb611246d5b214be1f657ea94bR49

Reply:
OK, then I will fix this. Do we have some task to fix this in PB?

Reply:
I don't think so.

Reply:
https://jira.ez.no/browse/EZEE-2542